fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml

```diff
@@ -113,6 +116,7 @@
                         </div>
                     </div>
                 </templates>
+                -->
```

Comment:
You can probably do something like it's done for project. Tasks in 12.0: https://github.com/odoo/odoo/blob/c881e3fe2a8b3db43fa9613a52b996fdfab21392/addons/project/views/project_views.xml#L621

Reply:
I did the change but I'm not sure that it is good because how to test it.